nexus/external-api/output/nexus_tags.txt

```diff
@@ -196,6 +196,9 @@ networking_bgp_status                    GET      /v1/system/networking/bgp-stat
 networking_loopback_address_create       POST     /v1/system/networking/loopback-address
 networking_loopback_address_delete       DELETE   /v1/system/networking/loopback-address/{rack_id}/{switch_location}/{address}/{subnet_mask}
```

Comment:
it does seem like including the rack-id in the path might make more sense based on this precedent. I'm not sure what @internet-diglett has in mind to wrangle the API, but I infer that's still in the works.